requirements.txt

```diff
@@ -1,6 +1,6 @@
 Django==1.8.4
 dj-database-url==0.3.0
-django-extensions==1.5.5
+django-extensions>=1.5.8
```

Comment:
I always pin exact versions in Python projects because writing I'm too lazy to write and don't like the look of `django-extensions>=1.5.8,<1.6` and Python doesn't have `1.5.x`, `^1.5.8` or `~1.5.8`.